### PR TITLE
add cndl

### PR DESCRIPTION
**Pull request by samisbakedham**
If not a bug:
Closes #4629 and #4853 

1-liner summary: Shows all changes above master, adds support for candle into alpha wallet

### CODE REVIEW

File: AlphaWallet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -6052,6 +6053,13 @@
 			path = Coordinators;
 			sourceTree = "<group>";
 		};
+		E7F44F672852BF69006A7C93 /* New Group */ = {
```
Comment (hboon) [inline AlphaWallet.xcodeproj/project.pbxproj:6056]: Would you help remove this and the other references to "New Group" that was added to the project file?

File: AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift [hunk 2]
```diff
@@ -67,6 +67,7 @@ struct TickerIdFilter {
         case .binance_smart_chain: return platform == "binance-smart-chain"
         case .avalanche: return platform == "avalanche"
         case .polygon: return platform == "polygon-pos"
+        case .candle: return platform == "candle"
```
Comment (hboon) [inline AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift:70]: I took a quick look, it doesn't look like candle is supported by CoinGecko, as in they don't provide data for tokens on Candle.

Would you help exclude this line too? We can add it again when there's data.

(CoinGecko does provide data for 0x3D97EdB1c1D87C0cBf098a0D2230d7380d4b1432 on Ethereum mainnet, that's fine, this line of code isn't about that, it's about tokens on Candle)

File: AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift [hunk 3]
```diff
@@ -83,6 +84,7 @@ struct TickerIdFilter {
         case .binance_smart_chain: return true
         case .avalanche: return true
         case .polygon: return true
+        case .candle: return true
```
Comment (hboon) [inline AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift:87]: Would you help make this false? Rationale like above (about CoinGecko)

**Comment by hboon** [comment]
Very close! Thanks for working with me here.

**Comment by samisbakedham** [comment]
> Very close! Thanks for working with me here.

Agreed very close! Will fix it now

File: AlphaWallet/Wallet/Models/InitialNetworkSelectionCollectionModel.swift [hunk 4]
```diff
@@ -9,7 +9,7 @@ import UIKit
 
 struct InitialNetworkSelectionCollectionModel {
 
-    static let defaultMainnetServers: Set<RPCServer> = [.main, .xDai, .polygon]
+    static let defaultMainnetServers: Set<RPCServer> = [.main, .xDai, .candle, .polygon]
```
[the remark this answers is not in the document]
Comment (samisbakedham) [reply AlphaWallet/Wallet/Models/InitialNetworkSelectionCollectionModel.swift:12]: Yes

File: AlphaWallet/Settings/Types/Constants.swift [hunk 5]
```diff
@@ -174,7 +175,7 @@ public struct Constants {
 
     static let ensContractOnMainnet = AlphaWallet.Address.ethereumAddress(eip55String: "0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85")
 
-    static let defaultEnabledServers: [RPCServer] = [.main, .xDai, .polygon]
+    static let defaultEnabledServers: [RPCServer] = [.main, .xDai, .candle, .polygon]
```
[the remark this answers is not in the document]
Comment (samisbakedham) [reply AlphaWallet/Settings/Types/Constants.swift:178]: Yes

File: AlphaWallet.xcodeproj/project.pbxproj (hunk 1 above)
Comment (samisbakedham) [reply AlphaWallet.xcodeproj/project.pbxproj:6056]: Yes

File: AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift (hunk 2 above)
Comment (samisbakedham) [reply AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift:70]: Yes

File: AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift (hunk 3 above)
Comment (samisbakedham) [reply AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift]: Fixed

**Comment by samisbakedham** [comment]
@hboon will fix now

**Comment by hboon** [comment]
Superseded by #5273 (for posterity)

**Pull request closed without merging**